lib/cn/kblock_reader.h

```diff
@@ -26,6 +27,17 @@ struct kblk_metrics {
     u32 tot_blm_pages;
 };
 
+/* TODO: seqno_min and seqno_max are not needed after the min and max seqno of a kvset
+ * is moved from the last kblock to a hblock
+ */
+struct kblock_desc {
+    struct cn            *cn;
+    struct kvs_mblk_desc *kd_mbd;
+    struct wbt_desc      *kd_wbd;
+    uint64_t              seqno_min;
+    uint64_t              seqno_max;
+};
```

Comment:
This struct is confusingly named since there's an entire family of functions that the term "kblock_desc" but don't use this struct (eg, kbr_get_kblock_desc).   I would remove the struct def and pass cn, mbd, and wbd around as params.  The seqno_min/max don't seem to be necessary, so just leave them out (see earlier comment).

Reply:
Or, what you said about putting wbd and mbd in kblock_desc.

Reply:
I would like to clean this up after Tristan's hblock changes are in. I'm in favor of leaving struct kblock_desc and fix other functions to take struct kblock_desc when referring to a kblock.

For now, I will remove seqno_min and seqno_max from this struct.